<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/JPEG siyahTurkce.jpg</normaloff>icons/JPEG siyahTurkce.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_7">
      <property name="statusTip">
       <string>Results</string>
      </property>
      <property name="title">
       <string>Results</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLabel" name="L_Output">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="3">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="statusTip">
         <string>Initial</string>
        </property>
        <property name="title">
         <string>Initial</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="B_Kmeans">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>K-Means</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="B_AffProp">
           <property name="text">
            <string>Aff Prop</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="B_MeanShift">
           <property name="text">
            <string>Mean Shift</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="B_Hierarcial">
           <property name="text">
            <string>Hierarchical</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="B_SpectralC">
           <property name="text">
            <string>Spectral Clustering</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="B_Dbscan">
           <property name="text">
            <string>DBSCAN</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGraphicsView" name="G_Initial">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGraphicsView" name="G_Final"/>
        </item>
       </layout>
      </item>
      <item row="0" column="2" rowspan="2">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="statusTip">
         <string>Final</string>
        </property>
        <property name="title">
         <string>Final</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="3" column="0">
          <widget class="QPushButton" name="B_Export_Final">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/173-1733262_insulating-an-existing-house-export-pdf-icon.png</normaloff>icons/173-1733262_insulating-an-existing-house-export-pdf-icon.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="B_Save_final">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/Paomedia-Small-N-Flat-Floppy.ico</normaloff>icons/Paomedia-Small-N-Flat-Floppy.ico</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="B_Clear_Final">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/Awicons-Vista-Artistic-Delete.ico</normaloff>icons/Awicons-Vista-Artistic-Delete.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="statusTip">
         <string>File</string>
        </property>
        <property name="title">
         <string>File</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="B_Open">
           <property name="statusTip">
            <string>Open</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/Avosoft-Warm-Toolbar-Folder-open.ico</normaloff>icons/Avosoft-Warm-Toolbar-Folder-open.ico</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="statusTip">
         <string>Final</string>
        </property>
        <property name="title">
         <string>Final</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="B_simulated">
           <property name="text">
            <string>Simulated</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="B_hillclimbing">
           <property name="text">
            <string>Hill Climbing</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="QGroupBox" name="groupBox_5">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="statusTip">
         <string>Initial</string>
        </property>
        <property name="title">
         <string>Ä°nitial</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="B_Save_initial">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/Paomedia-Small-N-Flat-Floppy.ico</normaloff>icons/Paomedia-Small-N-Flat-Floppy.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="B_Clear_inital">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/Awicons-Vista-Artistic-Delete.ico</normaloff>icons/Awicons-Vista-Artistic-Delete.ico</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="B_Export_initial">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>icons/173-1733262_insulating-an-existing-house-export-pdf-icon.png</normaloff>icons/173-1733262_insulating-an-existing-house-export-pdf-icon.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="groupBox_8">
      <property name="statusTip">
       <string>Manuel Solution</string>
      </property>
      <property name="title">
       <string>Manuel Solution</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="0">
        <widget class="QTextEdit" name="T_Points"/>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="L_Clusters"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Hubs</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="B_run">
         <property name="text">
          <string>RUN</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox_6">
      <property name="statusTip">
       <string>Info Panel</string>
      </property>
      <property name="title">
       <string>Info Panel</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTextBrowser" name="T_Output">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport_As">
     <property name="title">
      <string>Export As</string>
     </property>
     <addaction name="actionExportInitialSolution"/>
     <addaction name="actionExportFinal_Solution"/>
    </widget>
    <addaction name="actionOpen_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Initial_Soultion"/>
    <addaction name="actionSave_Final_Solution"/>
    <addaction name="separator"/>
    <addaction name="menuExport_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuClear">
     <property name="title">
      <string>Clear</string>
     </property>
     <addaction name="actionClearInitial_Solution"/>
     <addaction name="actionClearFinal_Solution"/>
    </widget>
    <addaction name="menuClear"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuClustering">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Clustering</string>
    </property>
    <addaction name="actionK_Means"/>
    <addaction name="actionAffinity_Propagation"/>
    <addaction name="actionMean_shift"/>
    <addaction name="actionSpectral_Clustering"/>
    <addaction name="actionHierarchical_Clustering"/>
    <addaction name="actionDBSCAN"/>
   </widget>
   <widget class="QMenu" name="menuHeuristics">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Heuristics</string>
    </property>
    <addaction name="actionHill_Climbing"/>
    <addaction name="actionSimulated_Anneling"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuClustering"/>
   <addaction name="menuHeuristics"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionK_Means">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>K-Means</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionAffinity_Propagation">
   <property name="text">
    <string>Affinity Propagation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionMean_shift">
   <property name="text">
    <string>Mean-shift</string>
   </property>
  </action>
  <action name="actionSpectral_Clustering">
   <property name="text">
    <string>Spectral Clustering</string>
   </property>
  </action>
  <action name="actionHierarchical_Clustering">
   <property name="text">
    <string>Hierarchical Clustering</string>
   </property>
  </action>
  <action name="actionDBSCAN">
   <property name="text">
    <string>DBSCAN</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSave_Initial_Soultion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Initial Soultion</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_Final_Solution">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Final Solution</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Shift+F4</string>
   </property>
  </action>
  <action name="actionClearInitial_Solution">
   <property name="text">
    <string>Initial Solution</string>
   </property>
  </action>
  <action name="actionClearFinal_Solution">
   <property name="text">
    <string>Final Solution</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionHill_Climbing">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hill Climbing</string>
   </property>
  </action>
  <action name="actionSimulated_Anneling">
   <property name="text">
    <string>Simulated Anneling</string>
   </property>
  </action>
  <action name="actionExportInitialSolution">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>InitialSolution</string>
   </property>
  </action>
  <action name="actionExportFinal_Solution">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Final Solution</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen_File</sender>
   <signal>triggered()</signal>
   <receiver>B_Open</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAffinity_Propagation</sender>
   <signal>triggered()</signal>
   <receiver>B_AffProp</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDBSCAN</sender>
   <signal>triggered()</signal>
   <receiver>B_Dbscan</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExportFinal_Solution</sender>
   <signal>triggered()</signal>
   <receiver>B_Export_Final</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExportInitialSolution</sender>
   <signal>triggered()</signal>
   <receiver>B_Export_initial</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_Final_Solution</sender>
   <signal>triggered()</signal>
   <receiver>B_Save_final</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_Initial_Soultion</sender>
   <signal>triggered()</signal>
   <receiver>B_Save_initial</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearFinal_Solution</sender>
   <signal>triggered()</signal>
   <receiver>B_Clear_Final</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClearInitial_Solution</sender>
   <signal>triggered()</signal>
   <receiver>B_Clear_inital</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionK_Means</sender>
   <signal>triggered()</signal>
   <receiver>B_Kmeans</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMean_shift</sender>
   <signal>triggered()</signal>
   <receiver>B_MeanShift</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSpectral_Clustering</sender>
   <signal>triggered()</signal>
   <receiver>B_SpectralC</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>564</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHierarchical_Clustering</sender>
   <signal>triggered()</signal>
   <receiver>B_Hierarcial</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
